### ISSUE TITLE
Search doesn't use all fields to look in

### ISSUE THREAD

**Issue by fly-man-** [monicahq/monica#2069]
It seems that the Search doesn't use fields like emailaddress to check if a person actually exists in the database.

Would be nice if those fields are included in the search

Example: Looking for Jack@beanstalk.com which comes up empty, however if I look for Jack Beanstalk the profile is found.

**Comment by alehkot** [comment]
+1, it would be great to have this feature.

**Comment by djaiss** [comment]
Indeed. This is on purpose. Right now we are looking at things that belong to the Contact object in the DB - not things like Contact Fields which are separate entities. It's not as easy as it seems to implement, but we want to implement it in the future for sure.

**Comment by djaiss** [comment]
Related somehow to #2079
